c++/src/Vector.cc

```diff
@@ -35,6 +35,7 @@ namespace orc {
                                           hasNulls(false),
                                           memoryPool(pool) {
     // PASS
```

Comment:
I was using "// PASS" to mean it was intentionally empty. Since you added a statement, you should remove the comment also.